model_compression_toolkit/quantizers_infrastructure/pytorch/inferable_quantizers/weights_inferable_quantizers/weights_symmetric_inferable_quantizer.py

```diff
@@ -0,0 +1,84 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import numpy as np
+
+from model_compression_toolkit.core.common.constants import FOUND_TORCH
+from model_compression_toolkit.quantizers_infrastructure.common.base_inferable_quantizer import QuantizationTarget
+
+if FOUND_TORCH:
+    import torch
+    from model_compression_toolkit.quantizers_infrastructure.pytorch.quantizer_utils import symmetric_quantizer, \
+    ste_round, ste_clip
+    from model_compression_toolkit.quantizers_infrastructure.pytorch.inferable_quantizers\
+        .base_symmetric_inferable_quantizer import \
+        BaseSymmetricInferableQuantizer
+
+    class WeightsSymmetricInferableQuantizer(BaseSymmetricInferableQuantizer):
+        """
+        Class for quantizing weights using a symmetric quantizer
+        """
+        def __init__(self,
+                     num_bits: int,
+                     threshold: torch.Tensor,
+                     signed: bool,
+                     per_channel: bool,
+                     ):
+            """
+            Initialize the quantizer with the specified parameters.
+
+            Args:
+                num_bits: number of bits to use for quantization
+                threshold: threshold for quantizing weights
+                signed: whether or not to use signed quantization
+                per_channel: whether to use per-channel quantization
+                channel_axis: axis along which to apply per-channel quantization
+            """
+
+            super(WeightsSymmetricInferableQuantizer, self).__init__(num_bits=num_bits,
+                                                                     threshold=threshold,
+                                                                     signed=signed,
+                                                                     quantization_target=QuantizationTarget.Weights)
+
+
+            self.per_channel = per_channel
+
+
+
+        def __call__(self, inputs, training=False):
+            """
+            Quantize the given inputs using the quantizer parameters.
+
+            Args:
+                inputs: input tensor to quantize
+                training: whether or not the quantizer is being used in training mode (unused here)
+
+            Returns:
+                quantized tensor.
+            """
+            inputs.requires_grad = False
+            w0 = ste_round(inputs / self.delta_tensor)
```

Comment:
Why you're using ste_round/clip in this quantizer but regular round/clip in the POT quantizer?